docker.gradle

```diff
@@ -0,0 +1,394 @@
+/*
+ * Copyright (C) 2007-2024 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import org.apache.commons.io.FilenameUtils
+
+buildscript {
+	repositories {
+		mavenCentral()
+	}
+	dependencies {
+		classpath group: 'commons-io', name: 'commons-io', version: project.property('commons-io.version')
+	}
+}
+
+// TODO: Use a Docker Gradle plugin to push up if possible
+// TODO: See https://plugins.gradle.org/search?term=docker
+
+ext {
+	imagesDir = "${projectDir}/resources/docker/images"
+	imagesBuildDir = "${buildDir}/images"
+	tempDir = "${buildDir}/temp"
+	libDir = "${tempDir}/lib"
+
+	authoringBuildTarget = 'main'
+
+	// TODO: Move this to EE
+	if (project.hasProperty("enterprise")) {
+		authoringBuildTarget = 'enterprise'
+	}
+
+	// Authoring bundle
+	authoringBundleSource = "${projectDir}/bundles/crafter-cms-authoring-${project.version}-linux-x86_64.tar.gz"
+	authoringBundleFile = "${tempDir}/" + FilenameUtils.getName(authoringBundleSource)
+	authoringBundleRoot = "${tempDir}/" + FilenameUtils.getBaseName(authoringBundleSource) + "/crafter"
+	// Delivery bundle
+	deliveryBundleSource = "${projectDir}/bundles/crafter-cms-delivery-${project.version}-linux-x86_64.tar.gz"
+	deliveryBundleFile = "${tempDir}/" + FilenameUtils.getName(deliveryBundleSource)
+	deliveryBundleRoot = "${tempDir}/" + FilenameUtils.getBaseName(deliveryBundleSource) + "/crafter"
+
+	defaultTomcatBinExcludes = [
+		"migration",
+		"upgrade",
+		"crafter-deployer",
+		"opensearch",
+		"dbms",
+		"cli",
+		"mongodb",
+		"crafter-cli",
+		"apache-tomcat/work/**",
+		"apache-tomcat/temp/**",
+		"apache-tomcat/webapps/crafter-search*",
+		"apache-tomcat/webapps/crafter-social*",
+		"apache-tomcat/webapps/crafter-profile*",
+		"apache-tomcat/webapps/ROOT",
+		"apache-tomcat/webapps/studio",
+		"**/*.pid"
+	]
+
+	defaultTomcatLocalBinExcludes = [
+		"migration",
+		"upgrade",
+		"mongodb",
+		"apache-tomcat/work/**",
+		"apache-tomcat/temp/**",
+		"apache-tomcat/webapps/crafter-search*",
+		"apache-tomcat/webapps/crafter-social*",
+		"apache-tomcat/webapps/crafter-profile*",
+		"apache-tomcat/webapps/ROOT",
+		"apache-tomcat/webapps/studio",
+		"**/*.pid"
+	]
+}
+
+task downloadLibSsl(type: Download) {
```

Comment:
@phuongnq is this still needed?

Reply:
@sumerjabri We don't need that for the `develop` branch where MariaDB 11 is used. I have sent PR for this (also update to Gradle 8.7): https://github.com/craftercms/docker-images/pull/93